jetstream/config.py

```diff
@@ -446,6 +500,50 @@ class MetricDefinition:
     bigger_is_better: bool = True
     analysis_bases: Optional[List[mozanalysis.experiment.AnalysisBasis]] = None
 
+    @staticmethod
+    def generate_select_expression(
+        param_definitions: Dict[str, ParameterDefinition],
+        select_expr_template: Union[str, jinja2.nodes.Template],
+    ) -> str:
+        """
+        Takes in param configuration and converts it to a select statement string
+        """
+
+        if "parameters" not in str(select_expr_template):
+            return _metrics_environment.from_string(select_expr_template).render()
+
+        select_expr = ""
+
+        for _param_name, _param_definition in param_definitions.items():
```

Comment:
Gotcha, I think that was my misunderstanding of when to use `_` as a prefix in var name. I assumed it could also be used to indicate a local scope of the variable.

Your definition makes sense, so I will try to follow that in the future.

Thanks! 🙇